dora/core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -801,6 +802,15 @@ public List<PageId> getCachedPageIdsByFileId(String fileId, long fileLength) {
     return pageIds;
   }
 
+  @Override
+  public void invalidateAllPagesByFileId(String fileId) {
```

Comment:
possible duplicate with the existing `invalidate` method.

Reply:
Renamed to deleteFile() as suggested.